src/cmd/cli/command/commands.go

```diff
@@ -223,6 +223,7 @@ func SetupCommands(ctx context.Context, version string) {
 	// Config Command (was: secrets)
 	configSetCmd.Flags().BoolP("name", "n", false, "name of the config (backwards compat)")
 	configSetCmd.Flags().BoolP("env", "e", false, "set the config from an environment variable")
+	configSetCmd.Flags().Bool("random", false, "set a random base64 value for config")
```

Comment:
What I suspect will be useful is a way to set the length of the random value, eg. `16` etc. There's a way to have flags that take an optional value, IIRC `configSetCmd.Flag("random").NoOptDefVal = "32"` (although I can see this being confusing since 32 can mean the length in chars or the length in bytes)

Reply:
Also not sure about "admitting" that it's Base64, since we remove the padding, so a regular *Standard* Base64 decoder can't actually decode it (unless we stick to a byte count multiple of 3, so we never get padding, eg. 30 or 33)